.github/actions/deploy/deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Deploy Simplenote'
+description: 'Builds and deploys Simplenote to VIP Go branches'
+branding:
+  icon: 'truck'
+  color: 'green'
+inputs:
+  APP_ID:
+    description: 'Application ID'
+    required: true
+  GITHUB_TOKEN:
+    description: 'GitHub Token'
+    requried: true
+  BRANCH:
+    description: 'Branch'
+    required: true
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+  entrypoint: './.github/actions/deploy/entrypoint.sh'
```

Comment:
the [docs](https://help.github.com/en/actions/building-actions/dockerfile-support-for-github-actions#entrypoint) indicate that this is here to override the `ENTRYPOINT` in the Dockerfile. since we're creating the Dockerfile and not overwriting anything I don't understand why we need it, or why we need the `entrypoint.sh` file at all.

it seems like if it works then things could be kept simpler if we directly embedded the call to `npm run deploy`

if it's the case that we need this external script in order to pass the `BRANCH` argument then I think a couple things would help:
 - remove `ENTRYPOINT` from the `Dockerfile`
 - add an explanatory comment why we're taking several steps in what seems like it should only take one

the docs don't seem incredibly clear to me. it would be worth verifying if we have access to the `ENV` vars like `BRANCH` from inside the Dockerfile without external helpers.

```Dockerfile
RUN dpkg…

CMD ["sh", "-c", "npm run deploy $BRANCH"]
```